rules/zig_binary.bzl

```diff
@@ -32,11 +32,11 @@ zig_binary = rule(
             allow_single_file = [".zig"],
         ),
         "_zig": attr.label(
-            default = "@zig_sdk//:tools/zig-wrapper",
+            default = "@zig_sdk-linux-amd64//:tools/zig-wrapper",
```

Comment:
Does this assume the execution platform to be always linux amd64?

Reply:
Yes, this was down for testing before I added the patch that creates the `zig_sdk` that auto-resolves which one is host for you, so will fix this up.

Reply:
If these are only used for local tests and not with RE, we can likely leave this be for now.